src/main/flight/pid.c

```diff
@@ -1126,7 +1123,7 @@ STATIC_UNIT_TESTED void applyItermRelax(const int axis, const float iterm,
 
         if (axis == FD_ROLL) {
             DEBUG_SET(DEBUG_ITERM_RELAX, 0, lrintf(setpointHpf));
-            DEBUG_SET(DEBUG_ITERM_RELAX, 1, lrintf(itermRelaxFactor * 100.0f));
+            DEBUG_SET(DEBUG_ITERM_RELAX, 1, lrintf(MAX(0, itermRelaxFactor * 100.0f)));
```

Comment:
Can `itermRelaxFactor` go negative? If so I don't understand the use of `MAX` here.

Reply:
The debug log is much easier to interpret when the magnitude of the cut as displayed can only go from 0-100, than say -279 to 100.  The code prevents negative numbers being used.

Reply:
Not really sure of this - I can see the motivation for it, considering `itermRelaxFactor` is only applied whenever it is > 0. But on the other hand we are 'wasting' valuable log space to write 0s, when we could be logging what `itermRelaxFactor` does when it is negative, and suppress printing the negative parts in the blackbox log viewer.

Reply:
We don't need to know how negative it goes, that's irrelevant.  BBE auto-scales the debugs, so if the minimum is always zero, and maximum is 100, the scaling will make logs from one craft to another readily comparable.  But the absolute lowest value can vary from flight to flight.  So then it gets really hard to see where zero is.  It could be ⅓ of the way from the top of the screen.  Much easier to interpret when zero is zero.

Reply:
Ok. In this case the functionality will become easier to understand if we cap `itermRelaxFactor` at a lower 0 in 1114, and then use it as it is.

Reply:
Does make the code more transparent, done.

Reply:
We don't need the `MAX()` in here now.